### PR TITLE
Parse and plot onionservice simulation data

### PR DESCRIPTION
**Pull request by sporksmith**
Splits simulation data by circuit-type (exit or onionservice), and generates separate graphs for each type where appropriate.

We omit the public Tor data (which is currently filtered to only exit circuits in `parse_onionperf.py`) from the onionservice graphs.

### CODE REVIEW

**Comment by sporksmith** [comment]
For reference, here is a comparison to the "golden data" before updating the golden data: https://github.com/shadow/tornettools/actions/runs/1814540081

File: tornettools/plot.py [hunk 1]
```diff
@@ -269,9 +305,9 @@ def __plot_client_goodput_5MiB(args, torperf_dbs, tornet_dbs):
 
     dbs_to_plot = torperf_dbs + tornet_dbs
 
-    __plot_cdf_figure(args, dbs_to_plot, 'client_goodput_5MiB',
+    __plot_cdf_figure(args, dbs_to_plot, 'client_goodput_5MiB.{circuittype}',
```
Comment (robgjansen) [inline tornettools/plot.py:308]: I think you want `f'client_goodput_5MiB.{circuittype}'` (i.e. you're missing the `f` prefix).

File: tornettools/plot.py [hunk 2]
```diff
@@ -192,9 +208,13 @@ def __plot_circuit_build_time(args, torperf_dbs, tornet_dbs):
 
     __plot_cdf_figure(args, dbs_to_plot, 'circuit_build_time',
```
Comment (robgjansen) [inline tornettools/plot.py:209]: I think you want `f'circuit_build_time.{circuittype}'` instead of just `'circuit_build_time'`.

**Review by robgjansen — APPROVED** [review]
Thanks for linking the artifacts - those helped me find 2 issues with the filenames :)

File: tornettools/plot.py (hunk 1 above)
Comment (sporksmith) [reply tornettools/plot.py]: Hah, done

File: tornettools/plot.py (hunk 2 above)
Comment (sporksmith) [reply tornettools/plot.py]: done

**Pull request merged**